test/README.md

```diff
@@ -83,8 +83,27 @@ Makefile targets
     test_results/compilable/json.d.out      runs an individual test
                                             (run log of the test is stored)
 
-In-test variables
+Test Configuration
 -----------------
+All tests defined within `.d` source files may be configured using the settings listed below, i.e.
+
+`mytest.d`:
+```
+/*
+REQUIRED_ARGS: -version=Foo
+TEST_OUTPUT:
+---
+Hello, World!
+---
+*/
+import std.stdio;
+void main(string[] args)
+{
+    writeln("Hello, World!");
+}
+```
+
+The most commonly used settings is probably `TEST_OUTPUT:`. The following is a list of all available settings:
```

Comment:
Oh I was just using this as a reasoning why it should be part of the example. Maybe just remove the first sentence, it sounds a bit informal?